Tests/MapboxCoreNavigationTests/NavigationServiceTests.swift

```diff
@@ -55,7 +55,7 @@ class NavigationServiceTests: XCTestCase {
     }
 
     func testDefaultUserInterfaceUsage() {
-        XCTAssertTrue(dependencies.navigationService.eventsManager.usesDefaultUserInterface, "MapboxCoreNavigationTests should have an implicit dependency on MapboxNavigation due to running inside the Example application target.")
+        XCTAssertTrue(!dependencies.navigationService.eventsManager.usesDefaultUserInterface, "MapboxCoreNavigationTests shouldn't have an implicit dependency on MapboxNavigation due to removing the Example application target as the test host.")
```

Comment:
At some point we should add a corresponding assertion that `userDefaultUserInterface` is true within MapboxNavigationTests.

Reply:
Thanks for the suggestion, I'll add a `TODO` note in this part to add the check in the MapboxNavigationTests.

Reply:
I add a corresponding test of `func testDefaultUserInterfaceUsage()` in the `MapboxNavigationTests.NavigationViewControllerTests.swift` when running inside the Example application target.